pkg/cmd/server/api/v1/conversions.go

```diff
@@ -309,6 +335,17 @@ func convert_runtime_RawExtension_To_runtime_Object(in *runtime.RawExtension, ou
 	return extension.Convert_runtime_RawExtension_To_runtime_Object(internal.Scheme, in, out, s)
 }
 
+// setDefault_ClientConnectionOverrides defaults a client connection to the pre-1.3 settings of
+// being JSON only. Callers must explicitly opt-in to Protobuf support in 1.3+.
```

Comment:
Correct.

On Mon, Jul 25, 2016 at 1:15 PM, David Eads <notifications@github.com>
wrote:

> In pkg/cmd/server/api/v1/conversions.go
> <https://github.com/openshift/origin/pull/9814#discussion_r72104795>:
>
> > @@ -309,6 +335,17 @@ func convert_runtime_RawExtension_To_runtime_Object(in *runtime.RawExtension, ou
> >  	return extension.Convert_runtime_RawExtension_To_runtime_Object(internal.Scheme, in, out, s)
> >  }
> >
> > +// setDefault_ClientConnectionOverrides defaults a client connection to the pre-1.3 settings of
> > +// being JSON only. Callers must explicitly opt-in to Protobuf support in 1.3+.
>
> TODO, we intent to swap this default in 1.4, correct?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9814/files/9aacfa7a5204e45bcbf3545f815774c42b2a5cde..1f31ddc8a6a04a2d760d460f604f0ff5f9cec074#r72104795>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABG_p6I7YVIlpXoT57PX1E-PgcCDVoj8ks5qZO85gaJpZM4JLFc7>
> .
>